example/src/components/Stats.tsx

```diff
@@ -0,0 +1,185 @@
+import {
+  Collapse,
+  CollapseHeader,
+  CollapseBody,
+} from 'accordion-collapse-react-native';
+import React, { useCallback } from 'react';
+import { View, StyleSheet, processColor } from 'react-native';
+import { LineChart } from 'react-native-charts-wrapper';
+
+import { Typo } from './Typo';
+
+type StatsProp = {
+  stats: any[];
+  label: string;
+};
+
+const notPlottableStats = [
+  'kind',
+  'rid',
+  'packetsLost',
+  'packetsReceived',
+  'bytesReceived',
+  'framesReceived',
+  'framesDropped',
+  'bytesSent',
+  'packetsSent',
+  'framesEncoded',
+];
+
+const chartDescriptionConfig = {
+  text: '',
+  textSize: 0,
+};
+
+const yAxisConfig = {
+  left: {
+    enabled: false,
+  },
+};
+
+export const Stats = ({ stats, label }: StatsProp) => {
+  const getValues = useCallback(
+    (label: string, chart: string) => {
+      const timestampsWithLabel = stats.filter((obj) => {
+        return Object.keys(obj).includes(label);
+      });
+
+      const values = timestampsWithLabel.map((obj) => {
+        return { y: obj[label][chart] !== null ? obj[label][chart] : 0 };
+      });
+
+      return values;
+    },
+    [stats]
+  );
+
+  const generateLimitationDurationInitData = useCallback(
+    (name: string, color: string) => {
+      return {
+        label: name,
+        values: [{ y: 0 }],
+        config: {
+          drawValues: false,
+          drawCircles: false,
+          color: processColor(color),
+        },
+      };
+    },
+    []
+  );
+
+  const getLimitationDurationsDataset = useCallback(
+    (label: string, chart: string) => {
+      const timestampsWithLabel = stats.filter((obj) => {
+        return Object.keys(obj).includes(label);
+      });
+
+      const combinedValues = timestampsWithLabel.map((obj) => {
+        return obj[label][chart] !== null ? obj[label][chart] : 0;
+      });
+
+      const res = [
+        generateLimitationDurationInitData('bandwidth', 'blue'),
+        generateLimitationDurationInitData('cpu', 'red'),
+        generateLimitationDurationInitData('none', 'green'),
+        generateLimitationDurationInitData('other', 'yellow'),
+      ];
+
+      combinedValues.forEach((obj) => {
+        res[0].values.push({ y: obj['bandwidth'] });
+        res[1].values.push({ y: obj['cpu'] });
+        res[2].values.push({ y: obj['none'] });
+        res[3].values.push({ y: obj['other'] });
+      });
+
+      return res;
+    },
+    [stats]
+  );
+
+  const getLineChart = useCallback((dataSets, isMultiValue: boolean) => {
+    return (
+      <LineChart
```

Comment:
can you make the chart line a tine bit bolder and in different color? Light blue is too bright to be readable

Reply:
Done.